am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +126,36 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * <p>
+     * Method requires name of service, the type of resource, name of resource and a role name to
+     * populate a  map of JsonPointers and set of permissions when record with user role and type exists in database.
+     * </p>
+     *
+     * @param serviceName  name of service
+     * @param resourceType type of resource
+     * @param resourceName name of a resource
+     * @param roleNames    A set of role names. Currently only one role name is supported but
```

Comment:
Do not leave dots at the end of line started with @param, @return or @throws.

Reply:
👍